Makefile

```diff
@@ -38,7 +38,14 @@ CONCORD_BFT_CMAKE_KEEP_APOLLO_LOGS?=TRUE
 CONCORD_BFT_CMAKE_BUILD_SLOWDOWN?=FALSE
 # Only useful with CONCORD_BFT_CMAKE_BUILD_TYPE:=Release
 CONCORD_BFT_CMAKE_BUILD_KVBC_BENCH?=TRUE
-
+# Only usefull with CONCORD_BFT_CMAKE_CXX_FLAGS_RELEASE=-O0 -g
```

Comment:
From my understanding with higher level of optimization we may lose some optimized out memory leaks (I have experienced it while playing with ASAN). So I decided that it would be better to analyze the code as is (better more info than less).
However, if you think it is critical to change to higher optimization level, we can try it.

Reply:
Let's leave it as is for now and see how it goes.